app/queues/certificate/order-completer-worker.server.ts

```diff
@@ -88,6 +97,15 @@ export const orderCompleterWorker = new Worker<CertificateJobData>(
     });
 
     logger.info(`Certificate ${certificateId} successfully issued and stored`);
+    // send success email notification
+    logger.debug(
+      `Sending success notification email for certificate with id=${certificateEntry.id}, username=${certificateEntry.username}, domain=${certificateEntry.domain}`
+    );
+    await addNotification({
+      emailAddress: certificateEntry.user.email,
+      subject: 'Attention: Certificate creation completed',
```

Comment:
Get rid of `Attention:`, it sounds like spam.  Make this: `My.Custom.Domain certificate ready`